configuration/floorplan.tcl

```diff
@@ -48,5 +49,6 @@ set ::env(RIGHT_MARGIN_MULT) 12
 
 set ::env(FP_HORIZONTAL_HALO) 10
 set ::env(FP_VERTICAL_HALO) $::env(FP_HORIZONTAL_HALO)
+set ::env(FP_PDN_ENABLE_GLOBAL_CONNECTIONS) 1
```

Comment:
I don't think we should keep it as an option since disabling it will crash the run. If there is no current usage, please remove and update OpenPDK's pdn configuration